DataRepo/loaders/sample_table_loader.py

```diff
@@ -394,7 +394,7 @@ def get_tissue(self, rownum, row):
                     MissingTissue(
                         tissue_name,
                         column=self.headers.TISSUE_NAME,
-                        rownum=row.name + 2,
+                        rownum=rownum + 1,
```

Comment:
nb. This will change in the pending refactor, so it's not that important.  The reason the error was occurring was because the method was expecting a row from `pd.iterrows()` (which should have been documented, but this was written before I started following that best practice), but instead, the loop is iterating on an enumeration of what was set as `sample_table_data = list(data)`.  The calculation of the row number was designed to be consistent with the way rows are reported in excel, where the headers are row 1.  This is the logic that is also applied to tsv and csv files, to be consistent.  So changing this to `rownum` here is correct, but the calculation of `rownum` was already done in line 279, so it should be (to be consistent with the rest of the codebase):
```suggestion
                        rownum=rownum,
```

Reply:
In fact, probably line 279 should be `+ 2`, now that I think about it. So I think you adding 1 was actually correct.

Reply:
I'm not sure about how the rest of the code works or what is expected, but using `+1` here did report the row number as one would expect when looking at the Excel file. IOW, Excel had `urine` on row 16 and this reported row 16.

Reply:
Yes, that's what I meant in my second comment with "So I think you adding 1 was actually correct.". It's just line 279 that should be updated.

Reply:
Thx. I"d say you go ahead and make any changes needed on this PR and just merge it.

Reply:
In that case, I'll just merge it.  Like I said, it's not that important and there's a refactor pending.